erpnext/projects/doctype/task/task.js

```diff
@@ -66,13 +66,13 @@ frappe.ui.form.on("Task", {
 	},
 
 	validate: function (frm) {
-		frm.doc.project && frappe.model.remove_from_locals("Project",
-			frm.doc.project);
+		frm.doc.default_project && frappe.model.remove_from_locals("Default_Project",
```

Comment:
@sahil28297 any idea what this does? (remove_from_locals)  also should Default_Project be Default Project??

Reply:
this removes it from the locals dict (cache)
```suggestion
		frm.doc.default_project && frappe.model.remove_from_locals("Default Project",
```

Reply:
@vishdha can you test this?